dts/bindings/mtd/eeprom-spi-i2c.yaml

```diff
@@ -8,6 +8,7 @@ include: eeprom-base.yaml
 properties:
     size:
       required: true
+      description: EEPROM size in bytes
```

Comment:
This is already present in `dts/bindings/mtd/eeprom-base.yaml`.

Reply:
Ok I will remove the whole property then

Reply:
No, leave it be. The `size` property is optional in `eeprom-base.yaml` but required in `eeprom-spi-i2c.yaml`.